### PR TITLE
Update Android gradle plugin and build tools

### PR DESCRIPTION
**Pull request by jaredsburrows**
@JakeWharton 
Updated android gradle plugin
Updated android build tools

### CODE REVIEW

File: telecine/build.gradle [hunk 1]
```diff
@@ -72,14 +72,14 @@ android {
 }
 
 dependencies {
-  compile 'com.android.support:support-annotations:23.2.1'
+  compile 'com.android.support:support-annotations:23.4.0'
```
Comment (JakeWharton) [inline telecine/build.gradle]: These changes are not related to build tools and android plugin and I would prefer they were done separately.

Comment (jaredsburrows) [inline telecine/build.gradle]: Should I make a separate pr to update "various deps"?

Comment (JakeWharton) [inline telecine/build.gradle]: You can make one PR, but I would prefer separate commits for each. Sometimes I'm not all that strict about it, but it makes doing things like `git bisect` a lot easier should there be a problem in the future.

Comment (jaredsburrows) [inline telecine/build.gradle]: Ok, give me a second.

Comment (jaredsburrows) [inline telecine/build.gradle]: @JakeWharton Fixed.

**Comment by JakeWharton** [comment]
:+1:
